tests/e2e/pageobjects/ide/DebugView.ts

```diff
@@ -41,6 +41,16 @@ export class DebugView {
         const runDebugButtonLocator: By = By.xpath('//span[@title=\'Start Debugging\']');
 
         await this.driverHelper.waitAndClick(runDebugButtonLocator);
+
+        await this.driverHelper.getDriver().wait(async () => {
+            Logger.debug(`Waiting for debugger to connect (threads to appear in "Threads" view)`);
+            const threadElements: WebElement[] = await this.driverHelper.getDriver().findElements(By.xpath(`//div[contains(@class, 'theia-debug-thread')]`));
+            if (threadElements.length < 2) {
+                return false;
```

Comment:
Have you tested this? Because from my experience the "wait" method will be exited successfully with any boolean value even if return "false". It would be great to check behavior with the negative variant.

Reply:
Yep... I've tried that and originally it was behaving as you are describing. But you had an error here: https://github.com/eclipse/che/pull/15643/files#diff-c4d939529680c0b7fe915783eda8a471R464
After this fixup it works as expected.

Reply:
anyway it may be simplified to:
```
await this.driverHelper.getDriver().wait(async () => {
            Logger.debug(`Waiting for debugger to connect (threads to appear in "Threads" view)`);
            const threadElements: WebElement[] = await this.driverHelper.getDriver().findElements(By.xpath(`//div[contains(@class, 'theia-debug-thread')]`));
            if (threadElements.length > 1) {
                return true;
            }
        });
```

Reply:
and one more suggestion, what about parametrization waiting by timeout:
```
await this.driverHelper.getDriver().wait(async () => {
            Logger.debug(`Waiting for debugger to connect (threads to appear in "Threads" view)`);
            const threadElements: WebElement[] = await this.driverHelper.getDriver().findElements(By.xpath(`//div[contains(@class, 'theia-debug-thread')]`));
            if (threadElements.length > 1) {
                return true;
            }
        }, timeout);
```